File: Examenes/EX2/22-1/BackEnd_CSharp/ConferenceSoftNETRemoting/ConferenceSoftModel/Ponencia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConferenceSoftModel
{
    public class Ponencia
    {
        private int _idPonencia;
        private Integrante _ponente;
        private String _titulo;
        private TimeSpan _horaInicio;
        private TimeSpan _horaFin;
        private bool _activo;
        public int IdPonencia { get => _idPonencia; set => _idPonencia = value; }
        public Integrante Ponente { get => _ponente; set => _ponente = value; }
        public string Titulo { get => _titulo; set => _titulo = value; }
        public TimeSpan HoraInicio { get => _horaInicio; set => _horaInicio = value; }
        public TimeSpan HoraFin { get => _horaFin; set => _horaFin = value; }
        public bool Activo { get => _activo; set => _activo = value; }
    }
}

File: Examenes/EX2/22-1/BackEnd_JAVA/ConferenceSoftModel/src/pe/edu/pucp/confesoft/model/Ponencia.java
package pe.edu.pucp.confesoft.model;
import java.io.Serializable;
import java.sql.Time;
public class Ponencia implements Serializable{
    private int idPonencia;
    private Integrante ponente;
    private String titulo;
    private Time horaInicio;
    private Time horaFin;
    private boolean activo;

    public int getIdPonencia() {
        return idPonencia;
    }

    public void setIdPonencia(int idPonencia) {
        this.idPonencia = idPonencia;
    }

    public Integrante getPonente() {
        return ponente;
    }

    public void setPonente(Integrante ponente) {
        this.ponente = ponente;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public Time getHoraInicio() {
        return horaInicio;
    }

    public void setHoraInicio(Time horaInicio) {
        this.horaInicio = horaInicio;
    }

    public Time getHoraFin() {
        return horaFin;
    }

    public void setHoraFin(Time horaFin) {
        this.horaFin = horaFin;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }
    
}